controllers/clusteroperator_test.go

```diff
@@ -266,3 +289,53 @@ func TestUpdateCOStatusDegraded(t *testing.T) {
 		}
 	}
 }
+
+func TestUpdateCOStatusAvailable(t *testing.T) {
```

Comment:
We are adding code to get the CO's name and version. Could we make sure they are set correctly in this test?

Reply:
Looks like this code will change significantly (and better). Thinking of closing this PR.